### DISCUSSION TITLE
Extension that allows me to click "return" in a function and be taken to the Caller of the function?

### DISCUSSION THREAD

**Discussion by JimNayzium** [microsoft/vscode-discussions#2395] [Extension Development QnA] [answered]
Does this exist?

The allows me to command-click a function and be taken to it, inside the function I want to click the "return" statement and be taken to the calling function of that function.

Surely this exists?

**Comment by alefragnani** [comment, marked as answer]
Hi @JimNayzium ,

I’m  not aware of any extension, nor setting, that would provide you such feature. So, if you really need such feature, I guess you would need to create a new extension.

Hope this helps
